### PR TITLE
[2.13.x] DDF-4008 Update pax web to 6.0.11 to match jetty version

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Updates pax web version to better work with the jetty version
#### Who is reviewing it? 
@kcover @emmberk @peterhuffer 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@lessarderic
@shaundmorris

#### How should this be tested?
Install DDF and do a basic smoke test on webapps
#### What are the relevant tickets?
[DDF-4008](https://codice.atlassian.net/browse/DDF-4008)

### CODE REVIEW

**Comment by clockard** [comment]
build now

**Comment by kcover** [comment]
Should we change my comment in the main pom saying that if any pax features need an update then all of pax-web should be updated, to avoid something like this happening in the future? atm it just says to overrided jetty if it needs an update.

**Comment by clockard** [comment]
build now

File: features/kernel/src/main/feature/feature.xml [hunk 1]
```diff
@@ -114,13 +61,11 @@
...
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
-    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="6.0.6_1">
+    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
         <feature version="[6.0,6.1)">pax-http</feature>
-        <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.4</bundle>
+        <bundle dependency="true" start-level="30">mvn:org.eclipse.jdt.core.compiler/ecj/4.5.1</bundle>
         <bundle start-level="30" dependency="true">mvn:javax.el/javax.el-api/3.0.0</bundle>
-        <!--<bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${project.version}</bundle>-->
-        <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-extender-whiteboard/6.0.6</bundle>
```
Comment (emmberk) [inline features/kernel/src/main/feature/feature.xml]: Why was this `pax-web-extender-whiteboard` line also removed? If it's not needed, it should be commented out and added to the comment at the top of the feature just like the `pax-web-jsp` line.

File: features/kernel/src/main/feature/feature.xml [hunk 2]
```diff
@@ -114,13 +61,11 @@
     </feature>
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
-    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="6.0.6_1">
+    <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
```
Comment (emmberk) [inline features/kernel/src/main/feature/feature.xml]: :+1: I reviewed this feature definiton with the original: https://github.com/ops4j/org.ops4j.pax.web/blob/web-6.0.11/pax-web-features/src/main/resources/features.xml#L124-L136/.

File: pom.xml [hunk 3]
```diff
@@ -149,7 +149,7 @@
         <spring.version>4.3.10.RELEASE</spring.version>
         <jetty.version>9.3.24.v20180605</jetty.version>
         <pax.cdi.version>0.12.0</pax.cdi.version>
-        <pax.web.version>6.0.9</pax.web.version>
+        <pax.web.version>6.0.11</pax.web.version>
```
Comment (emmberk) [inline pom.xml]: ✏️ Should the pax web version be tied with the jetty version? Would it be worthwhile to add a comment here?

File: features/kernel/src/main/feature/feature.xml [hunk 4]
```diff
@@ -114,13 +61,11 @@
...
         <feature version="[6.0,6.1)">pax-http</feature>
```
Comment (emmberk) [inline features/kernel/src/main/feature/feature.xml]: ❓ The `pax-http` feature is blacklisted, right? So this line won't do anything?

Comment (clockard) [reply features/kernel/src/main/feature/feature.xml]: only version 6.0.6 to 6.0.10 but not 6.0.11

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
connectivity issues
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**